static/js/components/PodcastFooter.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import { Link } from "react-router-dom"
+import moment from "moment"
+
+import { PODCAST_URL } from "../lib/url"
+
+export default function PodcastFooter() {
```

Comment:
Is it possible to merge some subcomponents of this with `Footer` to reduce duplication?

Reply:
the reason that I didn't is just that the layout requirements were *just* different enough that factoring out common stuff (beyond like trivial 3-line components) is more work than it saves

Reply:
so I'm sort of inclined to stick with a copy-paste approach here

Reply:
Sounds good to me